src/common/components/game/GameArea.js

```diff
@@ -0,0 +1,282 @@
+                isMyTurn={isMyTurn}
+                isAttackHappening={isAttackHappening}
+                onPassTurn={onPassTurn} />
+              <div style={{
+                display: 'flex',
+                justifyContent: 'space-around',
+                marginTop: 10
+              }}>
+                <SoundToggle />
+                <FullscreenToggle onClick={this.handleToggleFullScreen} />
+              </div>
+            </div>
+            <div
+              className="background"
+              style={{
+                display: 'flex',
+                justifyContent: 'center',
+                marginRight: 20,
+                zIndex: 9999
+              }}
+            >
+              <TutorialTooltip
+                tutorialStep={tutorialStep}
+                enabled={tutorialStep && tutorialStep.tooltip.location === 'endTurnButton'}
+                top={0}
+                place="left"
+                onNextStep={onNextTutorialStep}
+                onPrevStep={onPrevTutorialStep}
+              >
+                <EndTurnButton
+                  player={player}
+                  gameOver={gameOver}
+                  isMyTurn={isMyTurn}
+                  isAttackHappening={isAttackHappening}
+                  onPassTurn={onPassTurn} />
+              </TutorialTooltip>
+              <ForfeitButton
+                player={player}
+                history={history}
+                gameOver={gameOver}
+                isSpectator={isSpectator}
+                isTutorial={isTutorial}
+                onForfeit={onForfeit} />
+            </div>
+          </div>
+          <PlayerArea opponent gameProps={this.props} />
+          <div
+            className="background"
+            style={{
+              position: 'absolute',
+              left: 0,
+              top: 75,
+              bottom: 75,
+              right: 0,
+              margin: '0 auto',
+              zIndex: 999,
+              width: boardSize
+            }}
+          >
+            <Status status={isMyTurn ? status : {}} />
+            <Board
+              size={this.state.boardSize}
+              player={player}
+              currentTurn={currentTurn}
+              selectedTile={selectedTile}
+              target={target}
+              bluePieces={bluePieces}
+              orangePieces={orangePieces}
+              playingCardType={playingCardType}
+              tutorialStep={tutorialStep}
+              attack={attack}
+              isGameOver={!!winner}
+              onSelectTile={onSelectTile}
+              onActivateAbility={onActivateObject}
+              onTutorialStep={onTutorialStep}
+              onEndGame={onClickEndGame} />
+          </div>
+          <PlayerArea gameProps={this.props} />
+          <EventAnimation eventQueue={eventQueue} currentTurn={currentTurn} />
+          <VictoryScreen
+            winnerColor={winner}
+            winnerName={winner ? usernames[winner] : null}
```

Comment:
[nit] it might be nice here to replace the `null` with something like `usernames.None` that returns null for now. This would decouple the logic of no winner from our current implementation of it which is `null`.

This would introduce the edge case around a user named "None" though :\